tests/translator/input/embedded_connectors_depends_on_connector.yaml

```diff
@@ -0,0 +1,28 @@
+Resources:
+  MyFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Runtime: nodejs16.x
+      Handler: index.handler
+      InlineCode: |
+        exports.handler = async (event) => {
+          console.log(event);
+        };
+
+  MyQueue:
+    Type: AWS::SQS::Queue
+    Connectors:
+      MyConnector:
+        Properties:
```

Comment:
Missing tests with stuff next to `Properties`?

Reply:
Will be part of another PR where connector can support resource attributes